src/eterna/ui/MyCanvasResource.ts

```diff
@@ -0,0 +1,59 @@
+import {
+    ALPHA_MODES, BaseTexture, GLTexture, Renderer, Resource
+} from 'pixi.js';
+
+export default class MyCanvasResource extends Resource {
```

Comment:
There's probably a better name for this than `MyCanvasResource`.

Reply:
Actually, I don't think this is even used anywhere - it should probably just be removed